bookmarks.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bookmarks</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <header><h1>Bookmarks</h1></header>
+    <main>
+      <section>
+        <h2>Question:</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </p>
+        <details>
+          <summary>Show answer</summary>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua. At vero eos et accusam et justo duo dolores et ea
+          rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem
+          ipsum dolor sit amet.
+        </details>
+        <ul role="list">
+          <li>tag 1</li>
+          <li>tag 2</li>
+          <li>tag 3</li>
+          <li>tag 4</li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <nav>
+        <ul role="list">
+          <li>
+            <a href="./index.html"
+              ><img src="./images/home.png" alt="" width="40px" height="40px"
```

Comment:
Remember: no units on `width` and `height` of an `img`.
```suggestion
              ><img src="./images/home.png" alt="" width="40" height="40"
```

Reply:
Also: if possible, an SVG would be preferable to also look great on high resolution displays.

Reply:
Ok, now I got it :)